bundles/sirix-core/src/main/java/org/sirix/utils/IntToObjectMap.java

```diff
@@ -301,7 +314,7 @@ public void clear() {
     // And setting all the <code>next[i]</code> to point at
     // <code>i+1</code>.
     for (int i = 1; i < this.capacity; ) {
-      next[i] = ++i;
+      next[i] = i + 1;
```

Comment:
This is wrong, as it doesn't increment `i` anymore.

Reply:
What am I not seeing?

Reply:
Ah right it's not "I equals", my bad

Reply:
![image](https://github.com/sirixdb/sirix/assets/34094780/8639cd28-1331-438a-806d-0ee92480d58d)
![image](https://github.com/sirixdb/sirix/assets/34094780/5af0b774-4e55-4e48-84f2-34d05dc3834e)

Reply:
oh the increment isn't even present in the method

Reply:
Gonna keep an eye out for silly mistakes